### PR TITLE
VACMS-3629: automatically lint custom (s)css

### PR DESCRIPTION
**Pull request by acrollet**
## Description

See #3629 

## QA steps

As a developer
1. Review system theming
1. Then validate Acceptance Criteria from issue
- [ ] Linting rules make sense
- [ ] No functional/visual regressions are present

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [x] `Core Application Team`
- [ ] `Product Support Team`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [x] No announcement is needed for this code change. 
  - [x] Merge & carry on unburdened by announcements

### CODE REVIEW

**Comment by acrollet** [comment]
IDE integrations:

- https://www.jetbrains.com/help/phpstorm/using-stylelint-code-quality-tool.html
- https://marketplace.visualstudio.com/items?itemName=stylelint.vscode-stylelint

**Comment by acrollet** [comment]
I tuned down some of the pickiest rules, later on we could evaluate whether we feel it's worth getting more exact with things.

File: tests.yml [hunk 1]
```diff
@@ -36,6 +36,14 @@ va/tests/phpunit:
   description: PHPUnit
   command: phpunit --exclude-group disabled tests/phpunit --colors=always
 
+va/tests/module-lint-css:
```
Comment (indytechcook) [inline tests.yml:39]: Instead of running this on yaml-test, let's move these to github actions.

Comment (acrollet) [reply tests.yml:39]: cool, I like it. will do.

**Comment by indytechcook** [comment]
This is good stuff.  I'm trying to pull static code analysis out of the test suite which runs on the server into github actions so they can run in parallel. Can you also update https://github.com/department-of-veterans-affairs/va.gov-cms/blob/master/READMES/code-review.md as part of this PR?

File: .github/workflows/stylelint.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+name: Lint (s)css style
+
+on:
+  push:
+    branches: [ master ]
```
Comment (indytechcook) [inline .github/workflows/stylelint.yml:5]: Set this up to ignore master branch: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#example-ignoring-branches-and-tags

**Comment by acrollet** [comment]
Reviewdog integration has been added, linting issues fail the PR and are shown inline on the diff:

Screen Shot 2020-12-07 at 2 41 58 PM

**Comment by indytechcook** [comment]
I watched this commit on the `master` branchy and did not see the green checkbox.  I also checked the github actions log and the action did not run on master.

**Pull request merged**